velox/docs/functions/spark/datetime.rst

```diff
@@ -117,7 +117,8 @@ These functions support TIMESTAMP and DATE input types.
     date formatter in lenient mode that is align with Spark legacy date parser behavior or
     `Joda <https://www.joda.org/joda-time/>`_ date formatter depends on ``spark.legacy_date_formatter`` configuration.
     Returns NULL for parsing error or NULL input. When `Simple` date formatter is used, null is returned for invalid
-    ``dateFormat``; otherwise, exception is thrown. ::
+    ``dateFormat``; otherwise, exception is thrown. The `Simple` date formatter also permits partial date parsing;
```

Comment:
> partial date parsing

Would readers know what this means? Would be nice to clarify a bit and provide an example.

Reply:
@mbasmanova I've refined the documents and added function examples. I'm not sure if the examples meet the community standards. Could you please take a look?